test/integration/action/action-integration.spec.js

```diff
@@ -203,6 +210,19 @@ describe('Action Integration Tests', function() {
     });
   });
 
+  describe('Grpc and Settings actions', () => {
+    it('should get the locale', async () => {
+      const locale1 = await grpc1.getLocale();
+      expect(locale1, 'to be', 'gb');
+      settings1.setFiatByLocale({ locale: locale1 });
+      expect(store1.settings.fiat, 'to be', 'gbp');
+
+      const locale2 = await grpc2.getLocale(); // "Shona (Zimbabwe) - sn-ZW"
+      settings2.setFiatByLocale({ locale: locale2 });
+      expect(store2.settings.fiat, 'to be', 'usd');
+    });
+  });
+
```

Comment:
No need to add an integration test since we don't spin up electron here anyway. Unit tests should suffice for this feature.